src/modules/map/components/openlayers/OpenLayersVectorLayer.vue

```diff
@@ -46,16 +57,56 @@ export default {
                 style: this.styleUrl,
             },
         })
-        if (this.excludeSource) {
-            // we load the style on the side in order to be able to filter out some source
-            axios.get(this.styleUrl).then((response) => {
-                const vectorStyle = response.data
-                vectorStyle.layers = vectorStyle.layers.filter(
-                    (layer) => layer.source !== this.excludeSource
-                )
-                this.layer.maplibreMap.setStyle(vectorStyle)
-            })
-        }
+        this.setMapLibreStyle(this.styleUrl)
+    },
+    methods: {
+        setMapLibreStyle(styleUrl) {
+            if (this.excludeSource) {
+                // we load the style on the side in order to be able to filter out some source
+                axios.get(styleUrl).then((response) => {
+                    const vectorStyle = response.data
+                    vectorStyle.layers = vectorStyle.layers.filter(
+                        (layer) => layer.source !== this.excludeSource
+                    )
+                    this.mapLibreInstance.setStyle(vectorStyle)
```

Comment:
mapLibreInstance might be undefined !

Reply:
Good point, I will change all calls to `mapLibreInstance?.setStyle(...)`